frontend/src/components/TaskList.jsx

```
@@ -0,0 +1,144 @@
+          </button>
+        </div>
+
+        {tasks.length === 0 ? (
+          <div className={styles.noTasks}>No tasks yet. Create one to get started!</div>
+        ) : (
+          tasks.map((task) => (
+            <div
+              key={task._id}
+              className={`taskItem ${task.status === "Completed" ? "completed" : ""}`}
+              onClick={() => openEditTaskModal(task)}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)